packages/cli/src/scripts/add-files.ts

```
@@ -1,12 +1,51 @@
 import path from "path";
 import fs from "fs";
+import { fileURLToPath } from "node:url";
 import { Result } from "../types/registry.js";
 import { confirm, spinner } from "@clack/prompts";
 import { execSync } from "child_process";
 
-export const addFiles = async (framework: "nextjs" | "express" | "react" | "fastify" | "hono", provider: "dodopayments" | "stripe") => {
-    const result = await fetch(`https://billingsdk.com/tr/${framework}-${provider}.json`)
-        .then(res => res.json()) as Result;
+// packages/cli/src/types/registry.ts (excerpt)
+export type Result = {
```

Biome (2.1.2)

[error] Shouldn't redeclare 'Result'. Consider to delete it or rename it.

'Result' is defined here:


(lint/suspicious/noRedeclare)